contracts/Orchestrator.sol

```diff
@@ -42,7 +44,24 @@ contract Orchestrator is Ownable {
      */
     function rebase() external {
         require(msg.sender == tx.origin); // solhint-disable-line avoid-tx-origin
```

Comment:
We could just have one `rebase` function but add a modifier where we check if  either `msg.sender == tx.origin` or `msg.sender` is part of the whitelist